scilpy/samplingscheme/save_scheme.py

```diff
@@ -0,0 +1,90 @@
+from __future__ import division
+
+import logging
+import numpy as np
+
+# TODO: add some filename extension checking
```

Comment:
TODO? While we are migrating and cleaning, we should at least create an issue, else it will probably stay like this forever...